task/create_tables.sql

```diff
@@ -0,0 +1,50 @@
+use db
+go
+create table Bank
+(
+	Id int primary key identity(1,1),
+	Name nvarchar(50)
+)
+create table City
+(
+	Id int primary key identity(1,1),
+	Name nvarchar(50),
+	CitizenCount int
+)
+create table Filial
+(
+	Id int primary key identity(1,1),
+	BankId int not null foreign key references Bank(Id),
+	CityId int not null foreign key references City(Id),
+	Adress nvarchar(30) not null
+)
+create table Client
+(
+	Id int primary key identity(1,1),
+	Surname nvarchar(20),
+	Name nvarchar(20),
+	Patronymic nvarchar(20),
+	Age int check((age > 0) and (age >= 16))  not null,
+	PhoneNumber nvarchar(20)
+)
+create table SocialStatus
+(
+	Id int primary key identity(1,1),
+	Name nvarchar(20)
+)
+create table Account
+(
+	Id int primary key identity(1,1),
+	ClientId int not null foreign key references Client(Id),
+	FilialId int not null foreign key references Filial(Id),	
+	StatusId int not null foreign key references Social_status(Id),

```

Comment:
Before commit code, check all functionality
It's not enough just change all values, you should also check all code for any bugs like here